internal/osv2/mget.go

```diff
@@ -0,0 +1,152 @@
+package osv2
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/opensearch-project/opensearch-go/v2/opensearchapi"
+
+	"github.com/CrowdStrike/opensearchtools"
+)
+
+// The main purpose of this file is to implement the MGet interface on the OSv2Executor.
+// In order to do this, it also defines a bunch of marshalable types for making requests and getting
+// responses from OpenSearch 2. These might have a different shape than the similarily named ones in the root mget.
+
+// DoGet executes the Multi-Get MGetRequest using the provided opensearchtools.MGetRequest.
+// If the request is executed successfully, then a MGetResponse with MGetResults will be returned.
+// An error can be returned if
+//
+//   - The request to OpenSearch fails
+//   - The results json cannot be unmarshalled
+func (e *OSv2Executor) DoMget(ctx context.Context, req *opensearchtools.MGetRequest) (*opensearchtools.MGetResponse, error) {
```

Comment:
Could we get away with leaving off the `Do` prefix for the operation methods?  `MGet(...)` feels more intuitive than `GoMget()` to me.

Reply:
Ya, for sure.